src/CorePolicy.h

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2015-2017 Stanford University

```

Comment:
Copyright (c) 2017 ...

(should reflect date of first version of file; fix CorPolicy.cc also)